src/Android/Android.csproj

```diff
@@ -75,24 +75,25 @@
       <Version>2.1.0.4</Version>
     </PackageReference>
     <PackageReference Include="Portable.BouncyCastle">
-      <Version>1.8.10</Version>
+      <Version>1.9.0</Version>
     </PackageReference>
-    <PackageReference Include="Xamarin.AndroidX.AppCompat" Version="1.3.1.3" />
-    <PackageReference Include="Xamarin.AndroidX.AutoFill" Version="1.1.0.9" />
-    <PackageReference Include="Xamarin.AndroidX.CardView" Version="1.0.0.11" />
-    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.10" />
-    <PackageReference Include="Xamarin.AndroidX.MediaRouter" Version="1.2.5.2" />
-    <PackageReference Include="Xamarin.AndroidX.Migration" Version="1.0.8" />
+    <PackageReference Include="Xamarin.AndroidX.AppCompat" Version="1.5.1" />
+    <PackageReference Include="Xamarin.AndroidX.AutoFill" Version="1.1.0.13" />
+    <PackageReference Include="Xamarin.AndroidX.CardView" Version="1.0.0.16" />
+    <PackageReference Include="Xamarin.AndroidX.Core" Version="1.9.0" />
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.14" />
+    <PackageReference Include="Xamarin.AndroidX.MediaRouter" Version="1.3.1" />
+    <PackageReference Include="Xamarin.AndroidX.Migration" Version="1.0.10" />
```

Comment:
Do we still need the `Xamarin.AndroidX.Migration` package? are there still parts left to migrate to AndroidX?

Reply:
Edit 2: Okay it's gone, and _now_ I can say I believe everything has caught up since the app compiles and runs for me :)

Reply:
Great 😄